packages/test/test-service-load/src/utils.ts

```diff
@@ -302,6 +302,7 @@ export async function safeExit(code: number, url: string, runId?: number) {
 export const globalConfigurations: Record<string, ConfigTypes> = {
 	"Fluid.SharedObject.DdsCallbacksTelemetrySampling": 10000,
 	"Fluid.SharedObject.OpProcessingTelemetrySampling": 10000,
+	"Fluid.SharedTree.RebaseSampling": 10000,
```

Comment:
@yann-achard-MS 

Do you have a number in mind that you would like to have?

Reply:
@jikim-msft this is the one I'm saying we should remove, this is specifically for stress tests. The 1000 in editManager.ts is the one we should just make sure is at a value we like.

Reply:
Makes sense. Thanks for catching this. Would it be okay to create a constant variable inside the `editManager.ts` (e.g., `const THRESHOLD = 1000`) and pass that to the logger?

In this case, we can remove the monitoringContext.

Reply:
If we're very confident with the default it would be fine, but I'd say there's no harm in still using the configurable value from monitoringContext and just not overriding it anywhere, letting it be undefined there and default to 1000 or whatever we end up coalescing to in that line.

Reply:
I think that for production 1000 would be good (wherever that value is supposed to be)